cmd/ebrelayer/txs/relayToEthereum.go

```diff
@@ -13,93 +13,25 @@ import (
 	"github.com/Sifchain/sifnode/cmd/ebrelayer/types"
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/ethereum/go-ethereum/common"
-	ctypes "github.com/ethereum/go-ethereum/core/types"
+	ethereumtypes "github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/ethclient"
+
 	"go.uber.org/zap"
 )
 
 const (
 	// GasLimit the gas limit in Gwei used for transactions sent with TransactOpts
-	GasLimit            = uint64(500000)
+	GasLimit = uint64(500000)
```

Comment:
We need a higher gas limit here to allow for creation of the double pegged assets on destination chains. @daniportugalgit I would appreciate if you could post here how much it costs to create a new bridgetoken.

I think setting the gas limit at 2 million should be good and allow us to handle cases where bridgetokens are created.

Reply:
To create a new BridgeToken, we're currently using:
With 3 validators: 1,149,480 gas.
With 4 validators: 1,162,781 gas. (+13,301)
With 5 validators: 1,176,677 gas. (+13,896)
With 6 validators: 1,191,190 gas. (+14,513)
With 7 validators: 1,206,296 gas. (+15,106)
With 8 validators: 1,222,006 gas. (+15,710)

**Roughly**, starting from 3 validators, for each new validator we add, we'll spend an extra
`13,300 * numberOfNewValidators + 605 * (numberOfNewValidators - 1)` gas.

I agree that 2 million gas should be good. It would allow us to have up to 61 validators.